apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,14 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimFrame = document.createElement('iframe');
```

Comment:
Now I'm wondering what would be better/faster - approach like this, or having iframe in index.html and just do ```document.querySelector('.shim-host').src = '/views/shim_host.html?app_instance_id=' + App.instanceId;``` :)